src/app/query/bitcoin/balance/bitcoin-contracts-balance.query.ts

```diff
@@ -0,0 +1,25 @@
+import { UseQueryResult, useQuery } from '@tanstack/react-query';
+
+import { Money } from '@shared/models/money.model';
+
+import { useBitcoinContracts } from '@app/common/hooks/use-bitcoin-contracts';
+import { useCurrentAccountNativeSegwitIndexZeroSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+const staleTime = 1 * 60 * 1000;
+
+const balanceQueryOptions = {
+  staleTime,
+  keepPreviousData: false,
+  refetchOnMount: true,
+} as const;
+
+export function useBitcoinContractsBalanceQuery(): UseQueryResult<Money> {
+  const { sumBitcoinContractCollateralAmounts } = useBitcoinContracts();
+  const { address } = useCurrentAccountNativeSegwitIndexZeroSigner();
```

Comment:
This might break if we are using ledger and there is no bitcoin keys. Account index would work the same I think?

or do we only ever cal this when we know the are bitcoin keys?